### DISCUSSION TITLE
Trying to install via Pip on a new M2 Ventura MacBook but Erroring

### DISCUSSION THREAD

**Discussion by stfranklin525** [RhetTbull/osxphotos#815] [Q&A] [answered]
Hey there, 

Sorry if I'm missing something here but I'm trying to install this tool on my new Ventura MacBook with an M2 chip. With the command `pip3 install osxphotos`, the installation eventually fails. Here are a few potentially relevant bits from the error message:

```
Running setup.py install for pyobjc-core ... error
  error: subprocess-exited-with-error

  × Running setup.py install for pyobjc-core did not run successfully.
  │ exit code: 1
```

and 

```
Modules/objc/OC_PythonObject.m:241:18: error: variable 'cookie' set but not used [-Werror,-Wunused-but-set-variable]
          void*        cookie;
                       ^
      1 error generated.
      error: command '/usr/bin/clang' failed with exit code 1
      [end of output]
```

Any tips on how I can resolve this?

I'm running the latest version of python `3.11.0`

**Comment by RhetTbull** [comment, marked as answer]
> I'm running the latest version of python 3.11.0

I've not tested osxphotos with 3.11.0.  Pyobjc (which osxphotos requires) states it [works with 3.11.0](https://github.com/ronaldoussoren/pyobjc/releases) but again, I've not tested osxphotos and there may other dependencies. I suggest trying with python 3.10 until I can fully test osxphotos with 3.11.0.

**Reply by stfranklin525** [reply]
This did the trick, walking the version back to 3.10.0 and osxphotos installed successfully. Thank you!
